stardis/opacities/broadening.py

```diff
@@ -120,16 +120,16 @@ def calc_gamma_quadratic_stark(
     gamma_quadratic_stark : float
         Broadening parameter for quadratic Stark broadening.
     """
-    c4_prefactor = (ELEMENTARY_CHARGE**2 * BOHR_RADIUS**3) / (
-        36 * PLANCK_CONSTANT * VACUUM_ELECTRIC_PERMITTIVITY * atomic_number**4
+    c4_prefactor = (ELEMENTARY_CHARGE ** 2 * BOHR_RADIUS ** 3) / (
```

Comment:
Not a dealbreaker but I think no whitespace around `**` was making the whole equation more readable, because if I don't look carefully, my eyes see 2 getting multiplied to BOHR_RADIUS rather than `ELEMENTARY_CHARGE ** 2`

In one way you can think the existing white spaces as parentheses

Reply:
This was done by black. I happen to agree with you, but for right now black is the easiest way to have a consistent code style.